com.unity.multiplayer.mlapi/Runtime/Core/NetworkingManager.cs

```diff
@@ -1030,6 +1030,10 @@ private void HandleRawTransportPoll(NetEventType eventType, ulong clientId, Chan
         }
 
         private readonly BitStream m_InputStreamWrapper = new BitStream(new byte[0]);
+        bool m_InputStreamWrapperUsed;
+        // The fallback wrapper is used in case we have to handle incoming data but the InputStreamWrapper is already being used.
```

Comment:
Is it possible to nest > 2 levels deep?

Reply:
With the current MLAPI it's not possible. Reason being that the messages we buffer and execute in level 2 can never end up in another HandleIncomingData call. This is true because HandleIncomingData is only called in two cases:
1. When a new message arrives (nest level 1)
2. When that new message spawns an object and applies buffered messages (nest level 2)
Nest level 2 can never trigger case 1 or 2 again because case 1 can only be triggered by the server sending a spawn packet down and not locally and case 2 can only be triggered by case 1 specifically.

In the future we might introduce changes which change this in  the form of parallelizing the deserialization process. But then we would've had to rework the original code anyways and this change will make it obvious that there will be effort needed here to parallelize it properly.

Reply:
This all sounds great; can we hoist what you wrote into a code comment?   This seems to be valuable non-intuitive info worth keeping.

Reply:
I added a code comment explaining the above